src/Entity/Magazine.php

```diff
@@ -367,7 +367,7 @@ public function unsubscribe(User $user): void
 
     public function softDelete(): void
     {
-        $this->markedForDeletionAt = new \DateTime();
+        $this->markedForDeletionAt = date_add(new \DateTime(), new \DateInterval('P30D'));
```

Comment:
Functional it is doing the same, but personally I like this better
```suggestion
        $this->markedForDeletionAt = new \DateTime('now + 30days');
```
But I let you be the judge :)

EDIT: I just noticed that I did it the same way in my code :D